app/controllers/programmes_controller.rb

```diff
@@ -27,8 +34,12 @@ def sessions
           .map do |location|
             Session.new(team: current_user.team, location:, academic_year:)
           end
+    @unscheduled_sessions.sort_by! { |session| session.location.name }
```

Comment:
```suggestion
    @unscheduled_sessions.sort_by! { |session| session.location&.name }
```

Reply:
Ugh, apologies for leaving the inconsistencies in.

Should be fixed now everywhere now.